.github/workflows/test-demo.yaml

```
@@ -0,0 +1,60 @@
+name: Test VM demo
+
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
+  workflow_dispatch:
+
```

See [zizmor - excessive permissions](https://woodruffw.github.io/zizmor/audits/#excessive-permissions) audit rule

```suggestion

permissions: {}

```

---

I still think these security improvements need to be addressed, @imincik.

---

Fixed in c0a9a893f305a165d9601c4f897e68761a8dc482 .